core/src/main/java/org/apache/carbondata/core/preagg/TimeSeriesFunctionEnum.java

```diff
@@ -23,10 +23,15 @@
 public enum TimeSeriesFunctionEnum {
   SECOND("second", 0),
   MINUTE("minute", 1),
-  HOUR("hour", 2),
-  DAY("day", 3),
-  MONTH("month", 4),
-  YEAR("year", 5);
+  FIVE_MINUTE("five_minute", 2),
+  TEN_MINUTE("ten_minute", 3),
+  FIFTEEN_MINUTE("fifteen_minute", 4),
+  THIRTY_MINUTE("thirty_minute", 5),
```

Comment:
Any compatibility problem ? previous store was using hour=2, now new store thinks it as 'fiveminutes'

Reply:
no, because we always get the enum with name, so there wont be problem for compatibility, since timeseries was never used before, this will be the first time, so no problem